src/System.Net.HttpListener/tests/HttpRequestStreamTests.cs

```diff
@@ -77,9 +77,55 @@ public void Dispose()
         }
 
         [ConditionalTheory(nameof(PlatformDetection) + "." + nameof(PlatformDetection.IsNotOneCoreUAP))]
-        // [InlineData(true, "")] // [ActiveIssue(20246)] // CI hanging frequently
+        [InlineData(true, "")]
         [InlineData(false, "")]
-        // [InlineData(true, "Non-Empty")] // [ActiveIssue(20246)] // CI hanging frequently
+        [InlineData(true, "Non-Empty")]
+        [InlineData(false, "Non-Empty")]
+        public async Task Read_FullLengthAsynchronous_PadBuffer_Success(bool transferEncodingChunked, string text)
```

Comment:
This test appears to be failing on all runs with a cancellation exception:
Read_FullLengthAsynchronous_PadBuffer_Success(transferEncodingChunked: True, text: \"\")
```
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.ConfiguredTaskAwaitable`1.ConfiguredTaskAwaiter.GetResult()
   at System.Net.Http.HttpClient.<FinishSendAsyncBuffered>d__58.MoveNext() in /mnt/j/workspace/dotnet_corefx/master/portable-linux-Config_Debug+OuterLoop_false_prtest/src/System.Net.Http/src/System/Net/Http/HttpClient.cs:line 462
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at System.Net.Tests.HttpRequestStreamTests.<Read_FullLengthAsynchronous_PadBuffer_Success>d__6.MoveNext() in /mnt/j/workspace/dotnet_corefx/master/portable-linux-Config_Debug+OuterLoop_false_prtest/src/System.Net.HttpListener/tests/HttpRequestStreamTests.cs:line 118
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
```